src/components/aberration.tsx

```diff
@@ -1,56 +1,70 @@
-import React, { useEffect, useRef, useState } from 'react'
+import React, { useEffect, useRef, useState, FunctionComponent } from 'react'
 import { useWindowSize } from 'the-platform'
 import { aberrationRatio } from '../utils'
 
-const Aberration = ({ text }) => {
-    const ref = useRef(null);
-    const { height } = useWindowSize();
-    const [rect, setRect] = useState({top: 0, bottom: 0});
-    const [ratio, setRatio] = useState(0);
+const Aberration: FunctionComponent<{ text?: string }> = ({ text }) => {
```

Comment:
Is importing FunctionComponent from react a special typescript thing?

Reply:
Yep, that's the type for a function component. It used to be `SFC` but that name was deprecated since "stateless functional component" was kind of a misnomer, given function components can now have state, with hooks!